io.catenax.shared.business_partner_certificate/RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this model will be documented in this file.
+
+## [1.0.0] 23.02.2024
```

Comment:
you can change the date to 4th March if you need MS3 approval in next weekly meeting.

Reply:
Adapted date as 4th March